### PR TITLE
Fix doc rad plugin units and factors

### PR DESCRIPTION
**Pull request by PrometheusPi**
This pull request fixes inconsistencies in the radiation plugin documentation as reported in issue #3095.

### CODE REVIEW

File: docs/source/usage/plugins/radiation.rst [hunk 1]
```diff
@@ -515,9 +525,21 @@ The plugin supports multiple radiation species but spectra (frequencies and obse
...
+
+.. [Pausch2014]
+       Pausch, R., Debus, A., Widera, R. et al.
+       *How to test and verify radiation diagnostics simulations within particle-in-cell frameworks*
+       Nuclear Instruments and Methods in Physics Research, Section A: Accelerators, Spectrometers, Detectors and Associated Equipment, 740, 250–256 (2014)
+       https://doi.org/10.1016/j.nima.2013.10.073
+
+.. [Pausch2018]
+       Pausch, R., Debus, A., Huebl, A. at al.
+       *Quantitatively consistent computation of coherent and incoherent radiation in particle-in-cell codes—A general form factor formalism for macro-particles*
```
Comment (sbastrakov) [inline docs/source/usage/plugins/radiation.rst]: A space missing here `codes—A`

File: docs/source/usage/plugins/radiation.rst [hunk 2]
```diff
@@ -143,6 +146,7 @@ Form factor
 """""""""""
 
 The *form factor* is a method, which considers the shape of the macro particles when computing the radiation.
+More details can be found in [Pausch2018_.]
```
Comment (sbastrakov) [inline docs/source/usage/plugins/radiation.rst]: Shouldn't `[Pausch2018_.]` be `[Pausch2018_].` instead?

File: docs/source/usage/plugins/radiation.rst [hunk 3]
```diff
@@ -9,7 +9,10 @@ Our simulation computes the `Lienard Wiechert potentials <https://en.wikipedia.o
 
 .. math::
 
-   \frac{\operatorname{d}^2I}{\operatorname{d}{\Omega}\operatorname{d}\omega}\left(\omega,\vec{n}\right)=\left|\sum\limits_{k=1}^{N}\int\limits_{-\infty}^{+\infty}\frac{\vec{n}\times\left[\left(\vec{n}-\vec{\beta}_k(t)\right)\times\dot{\vec{\beta}}_k(t)\right]}{\left(1-\vec{\beta}_k(t)\cdot\vec{n}\right)^2}\cdot\operatorname{e}^{\operatorname{i}\omega\left(t-\vec{n}\cdot\vec{r}_k(t)/c\right)}\operatorname{d}t\right|^2
+   \frac{\operatorname{d}^2I}{\operatorname{d}{\Omega}\operatorname{d}\omega}\left(\omega,\vec{n}\right)= \frac{q^2}{16\pi^3\varepsilon_0 c} \left|\sum\limits_{k=1}^{N}\int\limits_{-\infty}^{+\infty}\frac{\vec{n}\times\left[\left(\vec{n}-\vec{\beta}_k(t)\right)\times\dot{\vec{\beta}}_k(t)\right]}{\left(1-\vec{\beta}_k(t)\cdot\vec{n}\right)^2}\cdot\operatorname{e}^{\operatorname{i}\omega\left(t-\vec{n}\cdot\vec{r}_k(t)/c\right)}\operatorname{d}t\right|^2
+
+Details on how radiation is computed with this plugin and how the plugin works in detail can be found in [Pausch2012]_.
```
Comment (sbastrakov) [inline docs/source/usage/plugins/radiation.rst]: `in detail` seems redundant, due to how the sentence starts.

**Comment by PrometheusPi** [comment]
@sbastrakov Thank you for the review - I will fix is a.s.a.p.

**Comment by PrometheusPi** [comment]
Ready for review

File: docs/source/usage/plugins/radiation.rst [hunk 4]
```diff
@@ -394,6 +398,12 @@ Dataset  Description                                           Dimensions
 ``z_Im`` imaginary part, z-component of the complex amplitude  (``N_observer``, ``N_omega``, 1)
 ======== ===================================================== ====================================
 
+.. note::
+
+   Please be aware, that despite the fact, that the SI-unit of each amplitude entry is :math:`\mathrm{[\sqrt{Js}]}`, the stored ``unitSI`` attribute returns :math:`\mathrm{[Js]}`.
+   This inconsistency will be fixed in the future.
```
Comment (ax3l) [inline docs/source/usage/plugins/radiation.rst]: When reading this I have no idea what to do as a user.
Should I multiply with `sqrt(unitSI)` to get the right unit? I am lost.

Comment (Anton-Le) [reply docs/source/usage/plugins/radiation.rst]: The statement is that the units of the _stored_ quantity **are** `[Js]` but **should actually be** `[\sqrt{Js}]` I would say that this is a hint to the user to take a square root of the _stored_ quantity. 

It is perhaps better to explicitly state that the user should perform the operation himself until the inconsistency is resolved.

Comment (PrometheusPi) [reply docs/source/usage/plugins/radiation.rst]: @ax3l I will find a more suitable wording tomorrow. @Anton-Le's explanation is correct.

Comment (ax3l) [reply docs/source/usage/plugins/radiation.rst]: Just state what a user must do with the values and the attribute in order to get the right result. Then note the inconsistency.

Comment (psychocoderHPC) [reply docs/source/usage/plugins/radiation.rst]: @PrometheusPi could you please fix this that we can merge it?

**Comment by PrometheusPi** [comment]
@psychocoderHPC please excuse the long delay - I now implemented the fix suggest by @ax3l and @Anton-Le 
It is added as the last commit.

**Comment by PrometheusPi** [comment]
@ComputationalRadiationPhysics/picongpu-maintainers feel free to review and merge

**Pull request merged**